cmd/spiffe-helper/config/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 	IncludeFederatedDomains  bool   `hcl:"include_federated_domains"`
 	RenewSignal              string `hcl:"renew_signal"`
 	DaemonMode               *bool  `hcl:"daemon_mode"`
+	EnableHealthCheck        *bool  `hcl:"enable_health_check"`
+	HealthCheckPort          int    `hcl:"health_check_port"`
```

Comment:
What do you think of putting these into a separate config block like what spire server does: https://github.com/spiffe/spire/blob/main/doc/spire_server.md#health-check-configuration